samples/bootloader/README.rst

```diff
@@ -15,68 +15,52 @@ See :ref:`ug_bootloader` for more information about the full bootloader chain.
 
 .. note::
 
-   Currently, this immutable bootloader does not support firmware updates over the SMP transport for either an upgradable bootloader or an application.
-   If the application using this bootloader requires SMP-based firmware updates, such as Bluetooth® LE DFU, :ref:`include MCUboot as a second-stage bootloader <ug_bootloader_adding_upgradable>`.
+   Currently NSIB does not support performing firmware updates over the SMP transport.
+   If the application using NSIB requires SMP-based firmware updates, such as Bluetooth® LE DFU, :ref:`include MCUboot as a second-stage bootloader <ug_bootloader_adding_upgradable>`.
 
 .. _bootloader_rot:
 
 Requirements
 ************
 
-The sample supports the following development kits:
+NSIB supports the following development kits:
 
 .. table-from-sample-yaml::
 
 Overview
 ********
 
-The sample accomplishes a simple and reliable *Root of Trust* (RoT) for a secure boot chain as follows:
+A simple and reliable *Root of Trust* (RoT) for a secure boot chain is implemented by NSIB as follows:
 
 1. Locks the flash memory.
 
-   To enable the RoT, the bootloader sample locks the flash memory that contains both the sample bootloader and its configuration.
-   Locking is done using the hardware that is available on the given architecture.
+   To enable the RoT the flash memory that contains itself and its configuration is locked using the hardware that is available on the given architecture.
 
    For additional details on locking, see the :ref:`fprotect_readme` driver.
 
-#. Selects the next stage in the boot chain.
+#. Selects the next slot in the boot chain.
 
-   The next stage in the boot chain can either be another bootloader or an application.
+   The next stage in the boot chain may either be an application or another bootloader and there may be one or two slots allocated for this stage.
+   Each slot has a version number attached to it and the slot with the latest version will be selected.
```

Comment:
```suggestion
   The next stage in the boot chain may either be an application or another bootloader, and one or two slots may be allocated for this stage. 
   Each slot has a version number, and the bootloader will select the slot with the latest version.

Reply:
Both suggested sentences from Sigvart look ok to me 👍🏻